compiler/AST/type.cpp

```diff
@@ -550,6 +550,12 @@ addDeclaration(AggregateType* ct, DefExpr* def, bool tail) {
       fn->addFlag(FLAG_METHOD_PRIMARY);
     }
   }
+
+  if (VarSymbol* var = toVarSymbol(def->sym)) {
+    // Identify VarSymbol as class/record member.
+    var->makeField();
```

Comment:
I'm sure there's some other part of the compiler that would treat a VarSymbol differently based on if it was a field or not and so could reasonably make use of this information when it is present instead of whatever it is doing presently, but it would be difficult to track it down so just using it here is the right decision.